data-info.properties.tmpl

```diff
@@ -0,0 +1,40 @@
+{{ with $v := (key (printf "%s/data-info/copy-key" $base))              }}data-info.copy-key                   = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/irods/max-paths-in-request" $base))      }}data-info.max-paths-in-request       = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/metadata/base" $base))                   }}data-info.metadata.base-url          = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/tree-urls/base" $base))                  }}data-info.tree-urls.base-url         = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/kifshare/flags/de-import" $base))        }}data-info.kifshare-download-template = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/data-info/kifshare-external-url" $base)) }}data-info.kifshare-external-url      = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/irods/admins" $base))                    }}data-info.perms-filter               = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/services/port" $base))                   }}data-info.port                       = {{ $v }}{{ end }}
+
+# ICAT configuration
+{{ with $v := (key (printf "%s/icat/host" $base))     }}data-info.icat.host     = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/icat/port" $base))     }}data-info.icat.port     = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/icat/user" $base))     }}data-info.icat.user     = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/icat/password" $base)) }}data-info.icat.password = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/icat/db" $base))       }}data-info.icat.db       = {{ $v }}{{ end }}
+
+# iRODS configuration
+{{ with $v := (key (printf "%s/irods/host" $base))        }}data-info.irods.host        = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/irods/port" $base))        }}data-info.irods.port        = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/irods/user" $base))        }}data-info.irods.user        = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/irods/pass" $base))        }}data-info.irods.password    = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/irods/base" $base))        }}data-info.irods.home        = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/irods/zone" $base))        }}data-info.irods.zone        = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/irods/resc" $base))        }}data-info.irods.resc        = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/irods/max-retries" $base)) }}data-info.irods.max-retries = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/irods/retry-sleep" $base)) }}data-info.irods.retry-sleep = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/irods/use-trash" $base))   }}data-info.irods.use-trash   = {{ $v }}{{ end }}
+{{ with $v := (key (printf "%s/irods/admin-users" $base)) }}data-info.irods.admin-users = {{ $v }}{{ end }}
+
+# file typing configuration
+data-info.type-detect.type-attribute = {{ (key (printf "%s/data-info/type-detect-attribute" $base)) }}
```

Comment:
I guess this one's required but the rest have defaults? Just making sure it's different for a reason, heh